modules/cnf-performance-profile-creator-arguments.adoc

```diff
@@ -74,6 +74,13 @@ Possible values:
 
 Default: `default`.
 
+| `per-pod-power-management`
+a|Enable Per Pod Power Management. This cannot be used with the ultra-low-latency power-consumption-mode.
```

Comment:
```suggestion
a|Enable per-pod power management. This cannot be used if you configured `ultra-low-latency` for the power consumption mode.
```

Reply:
My suggestion for above, following OpenShift doc guidelines to put parameters in backticks.